ptf/platform_helper/bfn_sai_helper.py

```diff
@@ -0,0 +1,20 @@
+import sys
+# from common_sai_helper import CommonSaiHelper
+sys.path.append("..")
+from  sai_base_test import *
+
+
+class BfnSaiHelper(common_sai_helper.CommonSaiHelper):
```

Comment:
imo it would be more clear if we used just a class name

Reply:
Good Point!